eslint.config.js

```diff
@@ -303,8 +305,8 @@ module.exports = [
       '**/mock*.{ts,tsx}',
     ],
     rules: {
-      '@grafana/no-untranslated-strings': ['error', { calleesToIgnore: ['^css$', 'use[A-Z].*'] }],
-      '@grafana/no-translation-top-level': 'error',
+      '@grafana/i18n/no-untranslated-strings': ['error', { calleesToIgnore: ['^css$', 'use[A-Z].*'] }],
+      '@grafana/i18n/no-translation-top-level': 'error',
```

Comment:
Something else we might want is to have a recommended config exported, that we spread onto here as well?

Reply:
yep think that would be smart - we can do that in a separate task